mcs/resource_mcs_kubernetes_cluster.go

```diff
@@ -183,12 +183,6 @@ func resourceKubernetesCluster() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-			"ingress_floating_ip": {
```

Comment:
you were right, we need it to read data from response, but remove `ForceNew` option.